source/topics/compliance/how-long-should-we-retain-audit-logs.md

```diff
@@ -0,0 +1,17 @@
+_Disclaimer: This material is provided for informational purposes only and not for the purpose of providing legal advice. Aptible is not a law firm. You should consult an attorney to obtain advice with respect to any particular issue or problem. Use of this site does not create an attorney-client relationship between you and Aptible._
+
+### This is a gray area, but the safest strategy is to retain HIPAA audit logs for 6 years. 
+
+The HIPAA Security Rule separates the requirement to implement audit logging from the requirement to retain documentation of your compliance program for 6 years.
+
+The logging requirement [45 C.F.R. § 164.312(b)] requires that regulated entities "[i]mplement hardware, software, and/or procedural mechanisms that record and examine activity in information systems that contain or use electronic protected health information."
+
+The retention requirement [45 C.F.R. 164.316(b)(1)(ii)] states, "[i]f an action, activity or assessment is required by [the Security Rule] to be documented, [you must] maintain a written (which may be electronic) record of the action, activity, or assessment."
+
+HHS has not clarified whether audit logs represent an "action, activity, or assessment" that falls within the scope of the retention requirement. In practice, many covered entities interpret the retention requirement as only applying to policies, procedures, and other more traditional documentation. These entities may only retain audit logs for a year or two.
+
+Some additional factors to consider:
+
+- Cheap, slow cloud storage such as Amazon Glacier (which is available for use under an AWS business associate agreement) can make a long-term audit log rotation and retention policy more cost-feasible than other options, such as retaining audit logs in a live database. 
```

Comment:
Totally up to you, but I wouldn't necessarily recommend using Glacier.

While storing audit data is definitely a good use case for it, Glacier requires quite a bit of expertise to use properly (or large sums of money), and I'd say it's not necessarily an ideal fit for the general public. Especially when talking about logs, where you'd  realistically only be saving a few bucks a month by using Glacier rather than the simpler alternative, S3.

So, you might consider recommending S3, possibly with archival to Glacier *on top of that* (S3 can automatically do this — it can *also* delete the object from S3 upon archiving to Glacier, but I would not recommend that): https://aws.amazon.com/blogs/aws/archive-s3-to-glacier/